village/src/components/SmurfForm.js

```diff
@@ -10,15 +11,15 @@ class SmurfForm extends Component {
     };
   }
 
-  addSmurf = event => {
+  addSmurf = (event, smurf) => {
     event.preventDefault();
-    // add code to create the smurf using the api
-
+    this.props.addSmurf(event, this.state)
     this.setState({
       name: '',
       age: '',
       height: ''
     });
+    window.location.reload();
```

Comment:
No need to reload anymore since the issue has been found. Instead,
```suggestion
    this.props.history.push("/")
```